### PR TITLE
Small changes to Bio.PDB

### PR DESCRIPTION
**Pull request by jgreener64**
On looking through the Bio.PDB module to get the hang of it I corrected a few style things that could be merged, though they are not essential:

- Correct some spelling
- Remove unused variables/imports
- Remove redundant brackets

### CODE REVIEW

File: Bio/PDB/PDBParser.py [hunk 1]
```diff
@@ -31,8 +31,7 @@
 class PDBParser(object):
     """Parse a PDB file and return a Structure object."""
 
-    def __init__(self, PERMISSIVE=True, get_header=False,
-                 structure_builder=None, QUIET=False):
+    def __init__(self, PERMISSIVE=True, structure_builder=None, QUIET=False):
```
Comment (peterjc) [inline Bio/PDB/PDBParser.py]: That looks like an API change (removing the ``get_header`` option) which is not backwards compatible.

File: Bio/PDB/QCPSuperimposer/__init__.py [hunk 2]
```diff
@@ -87,7 +87,7 @@ def set(self, reference_coords, coords):
         self.coords = coords
         n = reference_coords.shape
         m = coords.shape
-        if n != m or not(n[1] == m[1] == 3):
+        if n != m or not n[1] == m[1] == 3:
```
Comment (peterjc) [inline Bio/PDB/QCPSuperimposer/__init__.py]: Is this definitely equivalent? The comparison of three terms is in itself perhaps not the best style...

**Comment by peterjc** [comment]
With this second and larger contribution, are you happy to be thanked by name in the NEWS and CONTRIB file? Thanks!

**Comment by jgreener64** [comment]
Yes sounds good! Thanks for the merge.

**Pull request merged**